.github/workflows/ci.yaml

```diff
@@ -61,10 +61,10 @@ jobs:
           - 'git_push'
           - 'go_workspaces'
           - 'jest'
-          - 'oci_go_image'
```

Comment:
Is this change expected?

Reply:
yeah it was a duplicate line and was causing us to run that test twice